.github/workflows/test.migration.yaml

```diff
@@ -0,0 +1,127 @@
+name: "Test - Migration"
+on:
+  pull_request:
+    branches: [rc, main]
+  push:
+    branches: [rc, main]
```

Comment:
This workflow should not run on pushes to main. It will test twice against the same version.